articles/quickstart/native/ios-objc/_includes/_login_centralized.md

```diff
@@ -111,3 +111,9 @@ After the user authenticates, their information is returned in a `credentials` o
 ::: note
 To learn more about the `credentials` object, read the [Credentials](https://github.com/auth0/Auth0.swift/blob/master/Auth0/Credentials.swift) article.
 :::
+
+<%= include('../../../../_includes/_logout_url') %>
+
+::: note
+If you are following along with the sample project you downloaded from the top of this page, the logout URL you need to whitelist in the Allowed Logout URLs field is the same as the callback URL.
```

Comment:
This note is already included at the end of the include file. https://github.com/auth0/docs/blob/master/articles/_includes/_logout_url.md.